test/units/cli/test_galaxy.py

```diff
@@ -20,20 +21,30 @@
 __metaclass__ = type
 
 import ansible
+import json
 import os
+import pytest
 import shutil
 import tarfile
 import tempfile
 import yaml
 
+import ansible.constants as C
 from ansible import context
 from ansible.cli.arguments import option_helpers as opt_help
 from ansible.cli.galaxy import GalaxyCLI
-from ansible.errors import AnsibleError, AnsibleOptionsError
-from ansible.module_utils.six import PY3
+from ansible.errors import AnsibleError
+from ansible.module_utils._text import to_text
 from ansible.utils import context_objects as co
 from units.compat import unittest
-from units.compat.mock import call, patch
+from units.compat.mock import call, patch, MagicMock
+
```

Comment:
Are there tests for error handling of `ansible-galaxy collection install some_namespace.some_name:1.2.3.4` or other invalid version or version specifier formats?

Reply:
No, it just uses `LooseVersion` so it is quite liberal in what it supports.